mitodl/mitxonline-zendesk-theme
# style: update theme to match MITxOnline Site

## Related Ticket:
#3 
🚧 Rebased on #2 

## What this PR Does?
- Update the Zendesk theme to match the MITxOnline site
- Adds Contact Us block
- Adds header design changes
- Adds tweaks for updating hero design
- Updates the site logo
- Updates the header

## How to test

- Make sure that all the checkboxes mentioned in #3 should be completed in this PR
- You can visit https://mitxonline-test-theme.zendesk.com/hc/en-us to check this theme in a test instance.


## Screenshots
**Desktop**
![screencapture-mitxonline-test-theme-zendesk-hc-en-us-2021-12-15-19_03_45](https://user-images.githubusercontent.com/34372316/146201979-b05d5f85-972b-45fe-97b7-4e4b6c126895.png)

**Mobile**
![screencapture-mitxonline-test-theme-zendesk-hc-en-us-2021-12-15-19_04_09](https://user-images.githubusercontent.com/34372316/146202005-d46662fb-b24e-4faa-9b86-0361f225cc89.png)
